data/json/terrain.json

```diff
@@ -1269,7 +1275,8 @@
     },{
         "type" : "terrain",
         "id" : "t_reinforced_door_glass_o",
-        "name": "open reinforced  glass door",
+        "name": "open reinforced glass door",
+        "description": "A closed glass door reinforced with woven steel wires in a stylish, but practical pattern.  Yep, it's open.",
```

Comment:
A closed glass door ... is open. It sounds weird. "Unlocked" maybe?

Reply:
Yeah, looks like I had line 1253 and 1279 swapped. Corrected!